src/setMessageReaction.js

```diff
@@ -0,0 +1,75 @@
+"use strict";
+
+var utils = require("../utils");
+var log = require("npmlog");
+
+module.exports = function(defaultFuncs, api, ctx) {
+  return function setMessageReaction(reaction, messageID, callback) {
+    if(!callback) {
+      callback = function() {};
+    }
+    
+    switch (reaction) {
+      case "\uD83D\uDE0D": //:heart_eyes:
+      case "\uD83D\uDE06": //:laughing:
+      case "\uD83D\uDE2E": //:open_mouth:
+      case "\uD83D\uDE22": //:cry:
+      case "\uD83D\uDE20": //:angry:
+      case "\uD83D\uDC4D": //:thumbsup:
+      case "\uD83D\uDC4E": //:thumbsdown:
+      case "":
+        //valid
+        break;
+      case ":heart_eyes:":
+        reaction = "\uD83D\uDE0D";
+      case ":laughing:":
+        reaction = "\uD83D\uDE06";
+      case ":open_mouth:":
+        reaction = "\uD83D\uDE2E";
+      case ":cry:":
+        reaction = "\uD83D\uDE22";
+      case ":angry:":
+        reaction = "\uD83D\uDE20";
+      case ":thumbsup:":
+        reaction = "\uD83D\uDC4D";
+      case ":thumbsdown:":
+        reaction = "\uD83D\uDC4E";
+      default:
+        return callback({error: "reaction is not valid emoji"});
+        break;
+    }
+    
+    var variables = {
+      data: {
+        client_mutation_id: ctx.clientMutationId++,
```

Comment:
I think this will give NaN if ctx.clientMutationId is not initialized to a number.
You can fix that here: https://github.com/Schmavery/facebook-chat-api/blob/master/index.js#L58